src/account/SupportContact.tsx

```diff
@@ -25,6 +28,28 @@ import { currentAccountSelector } from 'src/web3/selectors'
 
 type Props = StackScreenProps<StackParamList, Screens.SupportContact>
 
+async function exportLogs() {
+  try {
+    Logger.showMessage(i18n.t('supportExportLogsStart'))
+    const logsPath = Logger.getReactNativeLogsFilePath()
+
+    // For now we need to export to a world-readable directory on Android
+    // TODO: use the FileProvider approach so we don't need to do this.
+    // See https://developer.android.com/reference/androidx/core/content/FileProvider
+    // and https://github.com/chirag04/react-native-mail/blame/340618e4ef7f21a29d739d4180c2a267a14093d3/android/src/main/java/com/chirag/RNMail/RNMailModule.java#L106
+    if (Platform.OS === 'android') {
+      const publicPath = `${RNFS.ExternalDirectoryPath}/${path.basename(logsPath)}`
+      await RNFS.copyFile(logsPath, publicPath)
+      return publicPath
+    }
+
+    return logsPath
+  } catch (e) {
+    Logger.showError(i18n.t('supportExportLogsError', { error: e }))
+    return false
```

Comment:
super nit: can you return `null` instead?  I think a return type of `null | string` is more conventional than `boolean | string`.  I realize this was probably based on existing code.

Reply:
Yes exactly, I kept the types expected by the other functions consuming the output of this.